src/site/content/en/blog/multi-screen-window-placement/index.md

```diff
@@ -168,106 +171,154 @@ The new `window-placement` permission can be queried with the
 ```js
 let granted = false;
 try {
-  const { state } = await navigator.permissions.query({ name: "window-placement" });
-  granted = state === "granted";
+  const { state } = await navigator.permissions.query({ name: 'window-placement' });
+  granted = state === 'granted';
 } catch {
   // Nothing.
 }
 ```
 
 The browser
 [can](https://webscreens.github.io/window-placement/#issue-c601b517:~:text=Permission%20check%20for%20above%3F,-partial)
```

Comment:
Please update this link, maybe https://webscreens.github.io/window-placement/#usage-overview-screen-information:~:text=This%20method%20may%20prompt%20the%20user%20for%20permission

Reply:
Done.